### PR TITLE
Fix(medusa-js/medusa) Fix type in AdminProductListTagsRes to use tags instead of types

### PR DESCRIPTION
**Pull request by adrien2p**
The actual type exposed a property `types` which is more link to the product-types. Instead it should expose a property `tags` otherwise you end up with a type error trying to access tags from the response type

### CODE REVIEW

**Review by olivermrbl — COMMENTED** [review]
Thanks @adrien2p ! Can I get you to change base branch to `develop` and rebase off of it? :)

**Comment by adrien2p** [comment]
Oups sorry though I did it ^^

File: packages/medusa-js/src/resources/admin/products.ts [hunk 1]
```diff
@@ -1,8 +1,4 @@
 import {
-  AdminGetProductsParams,
-  AdminPostProductsProductMetadataReq,
-  AdminPostProductsProductOptionsOption,
-  AdminPostProductsProductOptionsReq,
```
Comment (olivermrbl) [inline packages/medusa-js/src/resources/admin/products.ts:5]: These imports should not be deleted :)

**Review by olivermrbl — CHANGES_REQUESTED** [review]

File: packages/medusa-js/src/resources/admin/products.ts (hunk 1 above)
Comment (adrien2p) [reply packages/medusa-js/src/resources/admin/products.ts]: My bad wrong manip ;)

**Review by olivermrbl — APPROVED** [review]

**Pull request merged**